src/ml_tooling/transformers.py

```diff
@@ -246,3 +247,39 @@ def fit(self, X: pd.DataFrame, y=None):
     def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         data = self.scaler.transform(X)
         return pd.DataFrame(data=data, columns=X.columns, index=X.index)
+
+
+class DFRowFunc(BaseEstimator, TransformerMixin):
+    """
+    Row-wise operation on Pandas DataFrame. Strategy can either be one of
+    the predefined or a callable.    If some elements in the row is NaN these
+    elements are ignored for the build-in strategies.
+    """
+
+    func_map_ = {'sum': np.sum,
```

Comment:
To make a property "hidden", by convention we use starting underscore
```suggestion
    _func_map = {'sum': np.sum,
```

Alternatively, we can move this under __init__, (See refactor of FillNA), but here it shouldn't be a big problem